docs/topic-guides/web-apps/app-shell.md

```diff
@@ -19,6 +19,7 @@ The Starcounter app shell is a minimalist, configurable HTML document that impor
 *  the `<starcounter-include>` custom element
 *  Underwear.css, a CSS framework
 *  a placeholder for the Web App Manifest \(`link rel="manifest"`\).
+*  a small javascript that automatically adds the `aria-current-"page"` attribute to the links, which can be use e.g. to highligh the current link in the menu (Like [Uniform.css](https://docs.starcounter.io/v/2.4/topic-guides/blendable-web-apps/starcounter-design-system#uniform.css-ui-patterns-in-a-separated-presentation-layer) `uni-layout-left-nav` class does).
```

Comment:
> a small javascript that 

Something is missing here, right?

"attribute to the links" -> attribute to links

> which can be use e.g. to highligh the current link in the menu

Typo: "highlight", not "highligh". 

Typo: "can be use" -> "can be used"

Style: avoid "e.g", better: "which, for example, can be..."

Passive voice: "which can be use[d]" -> "which you can use"

Reply:
> (Like [Uniform.css]

"Like" should be lowercase. replace the parentheses with "menu, like Uniform.css...".

"like Uniform `un-layout-left-nav` class does" ->  "like the Uniform..."

Reply:
Take a look now

Reply:
That's alright, don't worry about it :)